### PR TITLE
Add simple README for Linux packages

### PR DESCRIPTION
**Pull request by yuya-oc**
Before submitting, please confirm you've
 - [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [x] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**
Added a README file for linux. It's included at the top directory of extracted packages. And it focuses on quick instruction for first use.

**Issue link**
#376 

**Test Cases**
N/A

**Additional Notes**
N/A

### CODE REVIEW

**Comment by jasonblais** [comment]
Looks great! I submitted a PR against your branch with a few suggested tweaks https://github.com/yuya-oc/desktop/pull/2

**Comment by yuya-oc** [comment]
@jasonblais Thanks! I added the final step, "Click **Save**".

**Comment by jasonblais** [comment]
Thanks! I fixed a typo on my PR.

I'll ask for a second review as it's good to have two reviewers on docs.

File: resources/linux/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+## Contributing
+
+See [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md) for reporting bugs, features or submitting pull requests.
+
+
+## License
+
+Apache Lisence, Version 2.0
```
Comment (lfbrock) [inline resources/linux/README.md]: Can we change "Apache Lisence" to "Apache License"?

**Comment by jasonblais** [comment]
Thanks!

@yuya-oc should be good to merge, thanks again for creating the file, this will be a great help for Linux installations.

**Comment by yuya-oc** [comment]
Okay, thanks for feedback!

**Pull request merged**